tests/test_broken_docs.py

```diff
@@ -77,7 +77,12 @@ def test_broken_links(test_app: SphinxTestApp):
 def test_broken_statuses(test_app: SphinxTestApp):
     test_app.build()
     assert get_warnings(test_app) == [
-        "<srcdir>/index.rst:11: WARNING: Need could not be created: Status 'NOT_ALLOWED' not in 'needs_statuses'. [needs.create_need]"
+        "ERROR: Need 'SP_TOO_002' has schema violations:",
```

Comment:
This makes SN more resilient, but the error also appears later in the build process, right?